python/cudf/cudf/_libxx/quantiles.pyx

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from cudf._libxx.lib cimport *
+from cudf._libxx.column cimport Column
+from cudf._libxx.table cimport Table
+cimport cudf._libxx.includes.quantiles as cpp_quantiles
+
+def quantiles(Table source_table,
+              vector[double] q,
+              object interp,
+              object is_input_sorted,
+              list column_order,
+              list null_precedence):
+    cdef table_view c_input = source_table.view()
```

Comment:
Since you are taking a full `view` and not passing specific column keys to cpp, will the index columns always be used in the quantiles calculation?  Perhaps it would be best to add an option to ignore the index (and just do something like: `c_input = source_table.data_view()`)

Reply:
@cwharris @rjzamora There's two situations with quantiles, neither of which depend on the index so we can always use `.data_view()` here. The first is that the quantile requested is a scalar float, in which case the index will be the column names of the input table. The second is the quantiles requested is a vector, in which case the index will be the vector of floats. Either way we can drop the index entirely 😄.

Reply:
Using `data_view` now, and I've added a method to `DataFrame` to create the appropriate new indexes.